modules/ROOT/pages/general-configuration-guide/attribution-requirements.adoc

```diff
@@ -14,6 +14,7 @@ If you are using {productname} on a paid tier of {cloudname} or have an active c
 
 == Free and Open Source Users
 
+// TODO: I think the link in the next line should be link:{legalpages}/attribution-requirements/[Terms of Service]
```

Comment:
You are correct that this should be a link. AsciiDoctor will technically resolve it as a link anyways (since the legalpages attributes starts with https), but it's hard to know it's a link and could break depending on the URL used so we made sure to include `link:` wherever attributes were used in the 6.x docs.

Reply:
The point I wanted to make is the url we are pointing toward is wrong, we use: `link:{legalpages}/tiny-cloud-services-subscription-agreement/` as opposed to: `link:{legalpages}/attribution-requirements/`

Reply:
Ahh right sorry... I can't answer that. I'd suggest logging a DOC jira for that instead of leaving a TODO here.

Reply:
@lnewson I made the suggested change by @EkimChau. Can you please mark the task as done? Thank you